hack/build/setup-pac-integration.sh

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
+set -x 
 
 PAC_NAMESPACE='openshift-pipelines'
 PAC_SECRET_NAME='pipelines-as-code-secret'
+SPRAYPROXY_NAMESPACE='sprayproxy'
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"/../..
```

Comment:
Why do you need that? It's not used anywhere.
It looks like you are trying to capture the Git top level directory. If that's the case `git rev-parse --show-toplevel` is shorter and not dependable to the script path.

Reply:
fixed.